opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_internal/metric_reader_storage.py

```diff
@@ -123,3 +123,73 @@ def collect(
                     )
 
         return metrics
+
+    def _handle_view_instrument_match(
+        self,
+        instrument: Instrument,
+        view_instrument_matches: List["_ViewInstrumentMatch"],
+    ) -> None:
+        for view in self._sdk_config.views:
+            # pylint: disable=protected-access
+            if view._match(instrument):
```

Comment:
I'd recommend like you did below to do a `continue` if they don't match so the rest of the body can be un nested
```suggestion
            if not view._match(instrument):
                continue
            # ...
```

Reply:
Good idea, fixed :+1: